inc/instance-config.php

```diff
@@ -459,54 +459,49 @@
-	),
-	'action' => 'reject',
-	'message' => "$fakereason_ano"
+    'condition' => array(
+        'filename'  => '/(TAKE ACTION v|trends.*associations|anusporn|anal insanity|anorectal risks|TAv[0-9]+|arisks)/', // Typical opening filename format. Their usual evasion strategy is to post only the image.
+    ),
+    'action' => 'reject',
+    'message' => "$fakereason_ano"
 );
 
 // Favorite names and buzzterms
 $config['filters'][] = array(
-	'condition' => array(
-		'body'  => '/(Rocco Siff|Evil Angel|Xavier Becerra|AdultDVDTalk|painal|Roughanal|anoreceptive|ltimately this is not about me|Logically-fallacious diversionary tactics)/',
-	),
-	'action' => 'reject',
-	'message' => "$fakereason_ano"
+    'condition' => array(
+        'body'  => '/(Rocco Siff|Evil Angel|Xavier Becerra|AdultDVDTalk|painal|Roughanal|anoreceptive|ltimately this is not about me|Logically-fallacious diversionary tactics)/',
+    ),
+    'action' => 'reject',
+    'message' => "$fakereason_ano"
 );
 
 /*
- * Filters for diverting TheThingN0ticer ban evader
+ * Filter TheThingN0ticer ban evader
  */
-$fakereason_thing = 'Due to automated child pornography and gore spam by /pol/, all posting now requires a pass.<br>To receive a one-week pass, email a short explanation of the Labor Theory of Value to space@national.shitposting.agency .';
-event_handler('post', function($post) {
-	$fakereason_thing = 'Due to automated child pornography and gore spam by /pol/, all posting now requires a pass.<br>To receive a one-week pass, email a short explanation of the Labor Theory of Value to space@national.shitposting.agency .';
-
-	// Detects posts in the /ITG/ with the filename "Untitled.png" and a Nazi flag
-	if (!$post->op && $post->board == 'leftypol' && $post->thread == 110463 && $post->has_file &&
-		$post->files[0]->filename == 'Untitled.png' &&
-		strpos($post->body_nomarkup, "Nazi</tinyboard>") !== false) {  /* has Nazi flag, hack */
-		return $fakereason_thing;
-    // Detects posts with the Nazi flag and their favorite Twitter links
-	} else if (strpos($post->body_nomarkup, "Nazi</tinyboard>") !== false && /* has Nazi flag, hack */
-		preg_match('/\/(WokeCapital|NickJFuentes|af_clips)/', $post->body)) {
-		return $fakereason_thing;	
-	}
+event_handler('post', function($post, $tor) {
+    if($post->board == 'leftypol'){
+    // note: just posting nazi flag with name doesn't trigger, on purpose
+    $n = 0;
+    // body is just a twitter account (or has ?lang=en or something)
+    if(preg_match('/^(https:\/\/)?(www.|m(obile)?.)?twitter\.com\/[a-zA-Z0-9_-]+\/?[#?&a-zA-Z0-9=_-]*(<tinyboard[^>]*>[^<]*<\/tinyboard>|<[^>]*>|\s)*$/',
+                    $post->body_nomarkup)){$n+=2;}
+    if($post->has_file && preg_match('/^Untitled[0-9]*.png/', $post->files[0]->filename)){$n+=2;}
+    if($post->name != 'Anonymous'){$n++; if($post->name == 'NasheedsSeedAndFeed'){$n+=2;}}
```

Comment:
perhaps here you could test if the IP has more than say 3 or 5 posts and instead of chasing their name, you could check if they are using a name other than Anonymous.

Reply:
This already does add a single point if they are using a non-Anonymous name (` if($post->name != 'Anonymous'){$n++;`), the hardcoded name simply adds two more points (to always trigger), which shouldn't ever be necessary, just a temporary insurance in the unlikely case they forget to attach an image and link.

I don't believe there is a current way to test the IP history but I don't believe it would be needed. I believe the conditions are loose enough that this wouldn't be necessary; it requires an extremely unusual filename or an extremely unusual post content, plus either the flag or non-Anonymous.